sdk/confidentialledger/azure-security-confidentialledger/src/main/java/com/azure/security/confidentialledger/ConfidentialLedgerClient.java

```diff
@@ -138,12 +140,12 @@ public Response<BinaryData> getEnclaveQuotesWithResponse(RequestOptions requestO
      * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
      * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
      * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
-     * @return a group of ledger collections along with {@link Response}.
+     * @return paginated collections returned in response to a query as paginated response with {@link PagedIterable}.
```

Comment:
nit: this might be due to a combination of autorest comment + swagger doc for paged APIs but it's a bit odd to mention "paginated" twice. Since this is generated, we can update the swagger doc (or if other languages don't have this issue, we can just add customization to fix this doc). For this release, we can just manually edit the comment in this PR.